examples/index.html

```diff
@@ -0,0 +1 @@
+<a href="./src/cube.html">cube.html</a><br><a href="./src/video.html">video.html</a><br><a href="./src/wireframe.html">wireframe.html</a><br>
```

Comment:
why is this here? Shouldn't the index.html be a) not comitted and b) in src/index.thtml?